packages/core/src/components/List/index.tsx

```diff
@@ -44,13 +48,38 @@ class List extends React.Component<Props & WithStylesProps> {
   }
 }
 
-export default withStyles(() => ({
+export default withStyles(({ unit }) => ({
   list: {
     listStyle: 'none',
     margin: 0,
     padding: 0,
   },
 
+  list_gutter: {
+    '@selectors': {
+      '> li': {
```

Comment:
`li + li`?

Reply:
that works too! this was handier for supporting horizontal + gutter